Add ROS 2 action CI for the bridge

evshary opened the pull request:
After supporting the topic and service, this is the last part to verify ROS 2 action communication with bridge.

JEnoch reviewed (changes requested):
For easier understanding of tests organisation, each filename should reflect the test it contains. e.g.: `action_client.rs` => `test_ros_client_zenoh_action.rs`.
For `topics.rs` that contain 2 tests, maybe `test_pub_sub.rs`.

I think the `_action/feedback` topic should also be tested. But that could come later in a separate PR.

evshary commented:
> For easier understanding of tests organisation, each filename should reflect the test it contains. e.g.: `action_client.rs` => `test_ros_client_zenoh_action.rs`. For `topics.rs` that contain 2 tests, maybe `test_pub_sub.rs`.

It makes sense to me. I've renamed the file.

> I think the `_action/feedback` topic should also be tested. But that could come later in a separate PR.

I agree with you. I've added a TODO comment in the code for the future update.
